jjb/dynamic/uperf.yml

```diff
@@ -0,0 +1,105 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+        # Disable logging
+        set +x
+        if [[ $SKIP_TLS_VERIFICATION ]]; then
+          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL} --insecure-skip-tls-verify
+        else
+          oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        fi
+        # Run tests
+        set -x
+        pushd workloads/network-perf
+        ./run_hostnetwork_network_test_fromgit.sh
+        ./run_pod_network_test_fromgit.sh
+        ./run_serviceip_network_test_fromgit.sh
+    concurrent: true
+    description: |
+      Uperf is a network performance tool
+      This job is managed by https://github.com/cloud-bulldozer/plow.git
+    disabled: false
+    name: RIPSAW-UPERF
+    node: scale-ci
+    parameters:
+    - string:
+        default: "kubeadmin"
+        description: User name to access cluster.
+        name: CLUSTER_USER
+    - string:
+        default: ""
+        description: Password for CLUSTER_USER to access cluster.
+        name: CLUSTER_PASSWORD
+    - string:
+        default: ""
+        description: The URL address to the openshift cluster to login to.
+        name: CLUSTER_API_URL
+    - string:
+        default: ""
+        description: The elasticsearch server where the data will be indexed.
+        name: ES_SERVER
+    - string:
+        default: ""
+        description: The port for the elasticsearch server.
+        name: ES_PORT
+    - bool:
+        default: true
+        description: Enables/Disables metadata collection
+        name: METADATA_COLLECTION
```

Comment:
We should include a `BASELINE_UPERF_UUID`, which we run our compare against

Reply:
done